<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="ServerButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>451</width>
        <height>331</height>
       </rect>
      </property>
      <property name="text">
       <string>Server</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ClientButton">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>100</y>
        <width>471</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>321</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>Hello enter path to configuration file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>521</width>
        <height>151</height>
       </rect>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>150</y>
        <width>201</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>wrong path</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>150</y>
        <width>431</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">label_2{
textsize: 30
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;server started&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1302</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
